mep/people/views.py

```diff
@@ -308,14 +308,18 @@ def get_context_data(self, **kwargs):
 
         # address of the lending library itself; automatically available from
         # migration mep/people/migrations/0014_library_location.py
-        library = Location.objects.get(name='Shakespeare & Company')
-        context['library_address'] = {
-            'name': library.name,
-            'street_address': library.street_address,
-            'city': library.city,
-            'latitude': str(library.latitude),
-            'longitude': str(library.longitude),
-        }
+        try:
+            library = Location.objects.get(name='Shakespeare and Company')
```

Comment:
I was thinking of using `first()` instead of try/except, but if you have to set all the info in context it probably doesn't make much difference.